src/main/java/processor/strategy/buffer/OperationBuffer.java

```diff
@@ -0,0 +1,7 @@
+package processor.strategy.buffer;
+
+public class OperationBuffer {
+    protected static String fruit;
+    protected static Integer operationAmount;
+    protected static Integer storedAmount;
+}
```

Comment:
that looks more like a model `FruitTransaction` class than the service, so let's do that